LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Refract Project
+Copyright (c) 2015 Mark W. Foster
```

Comment:
Copyright is for protecting authors, which I believe need to be legal entities. But frankly, just followed your convention on minim js :wink: